docs/docusaurus/i18n/en.json

```diff
@@ -288,6 +288,9 @@
       "lte/upgrade_debian_to_ubuntu": {
         "title": "Migrate AGWs to Ubuntu"
       },
+      "lte/GTPU_Ext_Header": {
+        "title": "GTP-U Extensions"
```

Comment:
It looks like this file is the old version since it has previous title. Could you please compile the docs again and update this?

Reply:
Fixed it.